master/buildbot/process/build.py

```diff
@@ -267,6 +267,8 @@ def getBuilderId(self):
                 tags = self.getProperty(
                     self.VIRTUAL_BUILDERTAGS_PROP,
                     self.builder.config.tags)
+                if '_virtual_' not in tags:
```

Comment:
you wll need to update the unit tests for this code

Reply:
Updated